cookbooks/wiki/templates/default/mw-ext-Translate.inc.php.erb

```diff
@@ -0,0 +1,36 @@
+<?php
+# DO NOT EDIT - This file is being maintained by Chef
+
+wfLoadExtension( 'Translate' );
+$wgGroupPermissions['user']['translate'] = true;
+$wgGroupPermissions['user']['skipcaptcha'] = true; // Bug 34182: needed with ConfirmEdit
+
+/**
+ * Language code for message documentation. Suggested values are qqq or info.
+ * If set to false (default), message documentation feature is disabled.
+ */
+$wgTranslateDocumentationLanguageCode = 'qqq';
+
+$wgGroupPermissions['autoconfirmed']['translate-messagereview'] = true;
+
+$wgGroupPermissions['translationadmin']['pagetranslation'] = true;
+$wgGroupPermissions['translationadmin']['translate-manage'] = true;
+
+/**
+ * Add a preference "Do not send me email newsletters" in the email preferences.
+ */
+$wgTranslateNewsletterPreference = true;
+
```

Comment:
```suggestion
/**
 * Add a preference "Do not send me email newsletters" in the email preferences.
 */
$wgTranslateNewsletterPreference = true;

```
This configuration variable removed in 2020. (MediaWiki 1.35)

https://github.com/wikimedia/mediawiki-extensions-Translate/commit/1c43244d7c56a24fe95cde1dec9a94cdfeceae69